.ci/travis.run

```diff
@@ -2,71 +2,40 @@
 
```

Comment:
This file should be changed to "ci.run" or something

Reply:
It should, I totally agree, but right now on master its called travis.run. I would take it as it is now, and then we can fixup all the names, across all the projects for uniformity next.